downloads.html

```diff
@@ -131,7 +131,12 @@ <h2>macOS</h2>
     <p>macOS 10.10 or later is recommended.</p>
 
     <ul>
-      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version <code>brew install minetest</code></li>
+      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version: 
+	<ul style="list-style: none;">
+	  <li><code>brew install minetest</code></li>
+	  <li><code>brew linkapps</code></li>		
+	</ul>
```

Comment:
Indents from `<ul` to `</ul>` contain tabs. Please use spaces. (I hate me for saying that)

Reply:
Done.